admin/class-purger.php

```diff
@@ -461,7 +461,11 @@ protected function do_remote_get( $url ) {
 		 */
 		do_action( 'rt_nginx_helper_before_remote_purge_url', $url );
 
-		$response = wp_remote_get( $url );
+		$request_args = array(
+			'blocking' => false,
+			'sslverify' => false
```

Comment:
This can be a security issue and `sslverify` should not be set to `false` at all unless you're testing things locally.